cookbooks/aws-parallelcluster-config/recipes/prep_env_compute_byos.rb

```diff
@@ -15,6 +15,8 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+include_recipe "aws-parallelcluster-config::mount_home"
```

Comment:
needs to go in the main prep_env no in the byos one otherwise we only mount home for byos scheduler